homeassistant/components/google/calendar.py

```diff
@@ -64,7 +70,12 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=15)
+# MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=15)
```

Comment:
Do we want to change the time between updates? Then we should remove this.

Reply:
I did not intent to change this. Reverted.